src/main/java/cloud/fogbow/rcs/core/service/CatalogService.java

```diff
@@ -32,24 +32,24 @@
     
     private static final Logger LOGGER = Logger.getLogger(CatalogService.class);
     
+    public static final String DOC_ENDPOINT = "/doc";
+    public static final String FORMAT_SERVICE_S_PORT_KEY = "%s_port";
+    public static final String FORMAT_SERVICE_S_URL_KEY = "%s_url";
+    public static final String KEY_SEPARATOR = "-";
     public static final String MEMBERSHIP_SERVICE_ENDPOINT = "/ms/members";
     public static final String PORT_SEPARATOR = ":";
+    public static final String SERVICE_ENDPOINT_FORMAT = "/rcs/service/%s/%s";
```

Comment:
i think this pattern was fixed recently. make sure to rebase this before merging.

Reply:
Fixed.